resources/js/views/Locations/Countries.vue

```diff
@@ -61,13 +61,13 @@
                     <div class="columns">
                         <div class="column is-half is-offset-3">
                             <div class="columns">
-                                <div class="column is-one-third">
-                                    <h1 class="subtitle is-5" style="color: black; text-align: center;">
+                                <div class="column has-text-centered">
+                                    <h1 class="subtitle is-6">
                                         <strong style="color:black;">
                                             {{ $t('location.maps10') }}
                                         </strong>
                                     </h1>
-                                    <h1 class="title is-2" style="text-align: center;">
+                                    <h1 class="title is-2">
```

Comment:
Why are we removing `center` here?

Reply:
We already have `has-text-centered` on the `div` tag.